src/features/defi/contexts/DefiManagerProvider/DefiCommon.ts

```diff
@@ -1,7 +1,7 @@
 import { ChainId } from '@shapeshiftoss/caip'
 
 export enum DefiType {
-  Pool = 'pool',
+  LiquidityPool = 'liquidity_pool',
```

Comment:
Do we need this change?

Reply:
@gomesalexandre the Products team wanted `liquidity pool` label, that's why I changed this.